.github/workflows/test-cloud-e2e.yml

```diff
@@ -13,7 +13,7 @@ on:
       - release-*
 
 env:
-  VCPKG_BINARY_SOURCES: 'clear;x-gha,readwrite'
+  VCPKG_BINARY_SOURCES: 'clear;'
```

Comment:
```suggestion
```

You can remove this evironment variable entirely.

Reply:
thanks, fixed!